start_integration_tests.py

```diff
@@ -0,0 +1,16 @@
+import argparse
+
+import tests_integration.tests_integration as tests_integration
+
+
+argument_parser = argparse.ArgumentParser(description='This program parses sequencing runs and uploads them to IRIDA.')
+argument_parser.add_argument('branch', help='Which IRIDA branch to run integration tests against')
+
+
+def main():
```

Comment:
Done.

Reply:
Sorry for not being clear, I meant more like file level comments about what this file is and how to run it.  I know in general this would be run by `make`, but since it could be done directly it would be nice to have some usage instructions in the file.

Reply:
Gotcha, added now.